go-controller/pkg/ovn/policy_test.go

```diff
@@ -1655,6 +1655,49 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Operations", func() {
 			err := app.Run([]string{app.Name})
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		})
+
+		ginkgo.It("can handle network policy with equivalent rules", func() {
+			app.Action = func(ctx *cli.Context) error {
+				namespace1 := *newNamespace(namespaceName1)
+				peer := knet.NetworkPolicyPeer{
+					IPBlock: &knet.IPBlock{
+						CIDR: "1.1.1.1",
+					},
+				}
+				// equivalent rules in one peer
+				networkPolicy1 := newNetworkPolicy(netPolicyName1, namespace1.Name, metav1.LabelSelector{},
+					[]knet.NetworkPolicyIngressRule{{
+						From: []knet.NetworkPolicyPeer{peer, peer},
+					}}, nil)
```

Comment:
nit: maybe consider encapsulating this into a properly named helper - that would allow you to drop the comment, which will help maintainability of the code (no need to keep the comment updated).

Reply:
So this test case has a very specific netpol config, that is why building the netpol is done manually. 
I find it actually more clear to see the internals of the netpol in this test, and comments are basically explaining what is being tested.
For most of the other tests with regular netpols functions like `getMatchLabelsNetworkPolicy` or `getPortNetworkPolicy` are used

Reply:
fair enough.